cirq/ops/pauli_string.py

```diff
@@ -26,11 +26,14 @@
 TDefault = TypeVar('TDefault')
 
 
-@value.value_equality
+@value.value_equality(manual_cls=True)
 class PauliString(raw_types.Operation):
-    def __init__(self,
-                 qubit_pauli_map: Mapping[raw_types.Qid, Pauli],
-                 coefficient: Union[int, float, complex] = 1) -> None:
+    def __init__(
+            self,
+            qubit_pauli_map: Optional[Mapping[raw_types.Qid, Pauli]] = None,
+            coefficient: Union[int, float, complex] = 1) -> None:
```

Comment:
This union type keeps re-appearing. WDYT about introducing it somewhere at the top level? I think `cirq.Scalar` may already exist...

Reply:
That scalar excludes integer. For now I am happy to list the cases.